R/update_ard.R

```diff
@@ -0,0 +1,60 @@
+#' Update ARDs
+#'
+#' Functions used to update ARD formatting functions and statistic labels.
```

Comment:
Perhaps also remind users that ARDs are just data frames, and they can directly modify it.

Reply:
added